src/routes/docs/tutorials/nextjs/step-5/+page.markdoc

```diff
@@ -0,0 +1,220 @@
+---
+layout: tutorial
+title: Add Database
+description: Add databases and queries to store user data in your Next JS app.
+step: 5
+---
+# Create collection {% #create-collection %} 
+In Appwrite, data is stored as a collection of documents. Create a collection in the [Appwrite Console](https://cloud.appwrite.io/) to store our expenses.
+
+{% only_dark %}
+![Create project screen](/images/docs/tutorials/dark/expense-tracker-collection.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/tutorials/expense-tracker-collection.png)
+{% /only_light %}
+
+In the collection go to attributes and add the following attributes:
+| Field       | Type    | Required | Elements       |
+|-------------|---------|----------|----------------|
+| userId      | String  | Yes      |                |
+| title       | String  | Yes      |                |
+| amount      | Integer | Yes      |                |
+| type        | Enum    | Yes      | Paid, Received |
+
+Now that we have created a collection, we have to add permissions to the collection. Go to the settings tab, scoll to the permissions sections add the following permissions:
+
+{% only_dark %}
+![Create project screen](/images/docs/tutorials/dark/collection-permissions.png)
```

Comment:
Please provide the original taken from a 1400x900 viewport with chrome/firefox devtools.

I will redo these with our branding assets.

Reply:
Added all the screenshots in 1400x900 viewport